FAQs.md

```diff
@@ -1,113 +1,135 @@
+Smart playlist (as the name implies) is a playlist that automatically sorts and adds music to the list according to customized rules / set parameters.
 
-4. **What is a smart playlist?**
 
-Smart playlist as the name implies is a playlist that automatically sorts and adds music to the list according to customized rules or parameters set. 
+5. **Can I import / export playlists?**
 
-5. **Can I import and export playlists on Jockey?**
+No, playlists can't be imported or exported. Apologies.
 
-No, playlists can neither be imported nor exported.
 
 6. **How can I add songs to a playlist?**
 
-Songs can be added to a playlist from any of the tabs. Songs, Artists, Albums, Genres. To add a song, simply click the 3 dots beside the song title and select the "Add to Playlist" option and select the desired playlist you wish the song to be added to.
+Songs can be added to playlists from any of the tabs (Songs, Artists, Albums, etc).
+To add a song, simply click the 3 dots beside the song title, select the "Add to Playlist" option, and select the desired playlist for said song.
 
-7. **Can I add highlight multiple songs at once to add a playlist?**
 
-No, the highlight function is not yet implemented on Jockey music player. However, when an artist, album or genre containing multiple songs are added to a playlist all the songs contained are added.
+7. **Can I highlight multiple songs at once to add to a playlist?**
+
+No, this has not been implemented yet. However, when an artist, album, or genre containing multiple songs are added to a playlist, all those songs are added.
+
 
 8. **Can I sort tracks?**
 
-Only tracks contained in a playlist can be sorted
+Only tracks contained in a playlist can be sorted.
+
+
+9. **What does the "include/exclude folder" function do?**
+
+This function allows you to include or exclude a folder from the music library.
+When a folder is excluded, the music files in it won't be listed in the music library.
+Note: Inclusion is the default for all folders. You must specify for a folder to be excluded for it not to be included.
 
-9. **What does include/exclude folder function do?**
-This function allows you to either include a folder a to be scanned and added to the music library. When a folder is excluded the music files in it won't be available in the music library.
 
 10. **How can I include or exclude a folder from my music library?**
 
-- Access Jockey player settings by clicking the 3 dots on the top right side of the app and selecting settings. 
-- Scroll down to library and select either to include or exclude. 
-- Click the plus(+) button and click on the folder you wish to include or exclude then click select.
+- Access the settings by clicking the 3 dots on the top-right side of the app (and hit "settings").
```

Comment:
This is actually changing in version 3.0 (it's being moved into a hamburger menu). Maybe just say open the in-app settings from the music library?